### ISSUE TITLE
Automatically generate [RequireExtension] for referenced projects

### ISSUE THREAD

**Issue by bricelam** [microsoft/vs-extension-testing#150]
Having to manually add it is rough--especially if you want to use the current Configuraiton (Debug or Release).

```cs
using Xunit.Harness;

#if DEBUG
[assembly: RequireExtension(@"..\..\..\..\..\VS.Data.Sqlite\bin\Debug\VisualStudio.Data.Sqlite.vsix")]
#else
[assembly: RequireExtension(@"..\..\..\..\..\VS.Data.Sqlite\bin\Release\VisualStudio.Data.Sqlite.vsix")]
#endif
```

...doing it in MSBuild is a bit better but still not great.

```xml
<ItemGroup>
  <AssemblyAttribute Include="Xunit.Harness.RequireExtension">
    <_Parameter1>..\..\..\..\..\VS.Data.Sqlite\bin\$(Configuration)\VisualStudio.Data.Sqlite.vsix</_Parameter1>
  </AssemblyAttribute>
</ItemGroup>
```

**Comment by sharwell** [comment]
I like the idea in theory, but a bunch of consumers of the project are using test configurations that rely on projects to deploy themselves (as part of the build), and don't use `RequireExtension` at all.

Note that it should be possible to copy the VSIX to the output directory and just have the filename listed. Here's one example:

https://github.com/tunnelvisionlabs/MouseFastScroll/blob/35cf9f43c87859d3215e8350af47c6ed756c2420/Tvl.VisualStudio.MouseFastScroll.IntegrationTests/Tvl.VisualStudio.MouseFastScroll.IntegrationTests.csproj#L29-L57

**Comment by bricelam** [comment]
Ah, I forgot they deploy on build. I was overthinking it.

**Comment by sharwell** [comment]
Yep, this can apply for all cases where you _only_ want to test against the same version/install of the IDE that you are building with. `RequireExtension` is valuable for other cases, e.g. where you want to build in VS 2022 but run your integration tests against VS 2019. Only the test harness knows how to run the VSIX installer for the version you are actually testing against when there is a mismatch.
